packages/ui/src/stories/molecules/Slider.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { Slider } from '../../components';
+
+const meta: Meta<typeof Slider> = {
+  component: Slider,
+  tags: ['autodocs'],
+};
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const Info: Story = {
+  args: {
+    captions: [
+      {
+        id: 1,
+        name: 'First ',
+      },
+      {
+        id: 2,
+        name: ' Second',
+      },
+    ],
+    min: 0,
+    max: 100,
+    decimal: 0,
+  },
+};
```

Comment:
Slider is too small and non-functional
![image](https://github.com/Cypherock/cypherock-cysync/assets/80461749/bbe9aba0-dab0-45d7-b2c7-b02e99f256ae)